frontend/src/app/Settings/Facility/facilitySchema.ts

```diff
@@ -0,0 +1,78 @@
+import * as yup from "yup";
+import { PhoneNumberUtil } from "google-libphonenumber";
+
+const phoneUtil = PhoneNumberUtil.getInstance();
+
+type PartialBy<T, K extends keyof T> = Omit<T, K> & Partial<Pick<T, K>>;
+
+export type RequiredFacilityFields = PartialBy<
+  Facility,
+  "id" | "email" | "streetTwo" | "city"
+>;
+
+type RequiredProviderFields = PartialBy<
+  Provider,
+  "middleName" | "suffix" | "phone" | "streetTwo" | "city"
+>;
+
+const providerSchema: yup.SchemaOf<RequiredProviderFields> = yup.object({
+  firstName: yup.string().required("Ordering provider first name is missing"),
+  middleName: yup.string(),
+  lastName: yup.string().required("Ordering provider last name is missing"),
+  suffix: yup.string(),
+  NPI: yup.string().required("Ordering provider NPI is missing"),
+  phone: yup.string(),
+  street: yup.string().required("Ordering provider street is missing"),
```

Comment:
The Ordering Providers address is not required. `street`, `state`, and `zipCode` should be optional

Reply:
....turns out nothing about the ordering provider is required